pkg/util/ipset/ipset_test.go

```diff
@@ -502,6 +501,54 @@ func TestTestEntry(t *testing.T) {
 	}
 }
 
+func TestTestEntryIPv6(t *testing.T) {
+	testEntry := &Entry{
+		IP:       "fd00:1234:5678:dead:beaf::1",
+		Port:     8080,
+		Protocol: ProtocolTCP,
+		SetType:  HashIPPort,
+	}
+	setName := "NOT"
+	fcmd := fakeexec.FakeCmd{
+		CombinedOutputScript: []fakeexec.FakeAction{
+			// Success
+			func() ([]byte, []byte, error) {
+				return []byte("fd00:1234:5678:dead:beaf::1,tcp:8080 is in set " + setName + "."), nil, nil
+			},
+			// Failure
+			func() ([]byte, []byte, error) {
+				return []byte("fd00:1234:5678:dead:beaf::1,tcp:8080 is NOT in set FOOBAR."), nil, &fakeexec.FakeExitError{Status: 1}
```

Comment:
you should change the IP here to a nonexistant one

Reply:
the test doesn't seem to check it but is confusing, because it actually exists

Reply:
Thank you ! I modified it to nonexistent one.